launcher/launcher.go

```diff
@@ -1394,3 +1399,30 @@ func (launcher *Launcher) handleTTLs(ctx context.Context) {
 		}
 	}
 }
+
+func (launcher *Launcher) stopRunningInstances() error {
+	log.Debug("Stop running instances")
+
+	instances, err := launcher.storage.GetRunningInstances()
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	launcher.getCurrentServices(instances)
+
+	launcher.currentInstances = make(map[string]*instanceInfo)
+
+	// Fill launcher.currentInstances
+	_ = launcher.getStartInstances(instances)
+
+	// Set current instances as started
+	for _, instance := range launcher.currentInstances {
+		instance.isStarted = true
```

Comment:
maybe better to set inside `getStartInstances ` ?

Reply:
Normally isStarted is set in startInstance function.  But this function is not called in this case. I will redesign this part.